### ISSUE TITLE
contribute.json disappeared with our static file switch

### ISSUE THREAD

**Issue by ghost** [mozilla/testpilot#1582]
https://testpilot.firefox.com/contribute.json  .  Can probably close #1507 with this as well.

**Comment by pdehaan** [comment]
I think this may have been fixed via #2248.

#### https://testpilot.dev.mozaws.net/contribute.json:

```js

{
    "name": "Test Pilot",
    "description": "Test Pilot is a platform for performing controlled tests of new product concepts in Firefox.",
    "repository": {
        "url": "https://github.com/mozilla/testpilot",
        "license": "MPL2",
        "tests": "https://circleci.com/gh/mozilla/testpilot"
    },
    "participate": {
        "home": "https://wiki.mozilla.org/Test_Pilot",
        "docs": "https://github.com/mozilla/testpilot/blob/master/README.md",
        "irc": "irc://irc.mozilla.org/#testpilot",
        "mailing-list": "https://mail.mozilla.org/listinfo/testpilot-dev"
    },
    "bugs": {
        "list": "https://github.com/mozilla/testpilot/issues",
        "report": "https://github.com/mozilla/testpilot/issues/new"
    },
    "urls": {
        "prod": "https://testpilot.firefox.com",
        "stage": "https://testpilot.stage.mozaws.net",
        "dev": "https://testpilot.dev.mozaws.net"
    },
    "keywords": [
        "css",
        "javascript",
        "react",
        "redux",
        "sass"
    ]
}
```

---

#### https://testpilot.dev.mozaws.net/__version__:

```js
{
  commit: "bdf422a5b71559febdaee43dd7db0ae91a713f5f",
  version: "",
  source: "https://github.com/mozilla/testpilot",
  build: "https://circleci.com/gh/mozilla/testpilot/3515"
}
```

**Comment by ghost** [comment]
Looks like it!  Thanks  https://testpilot.firefox.com/contribute.json
